front/src/components/login/LoginPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { loginUserAction } from '../../common/redux/actions/authActions';
+import { Redirect } from 'react-router-dom';
+import { LoginForm } from './LoginForm';
+
+class LoginPage extends Component {
+  onHandleLogin = (data) => {
+    this.props.dispatch(loginUserAction(data));
+  }
+  
+  componentDidMount() {
+    document.title = 'Login';
```

Comment:
We don't need this line. The name of the browser tab should not change.

Reply:
done